operator/validator/controller.go

```diff
@@ -1131,6 +1139,7 @@ func (c *controller) UpdateValidatorMetaDataLoop() {
 	const batchSize = 512
 	var sleep = 2 * time.Second
 
+	activeSubnets := c.network.ActiveSubnets()
```

Comment:
for performance reasons, because this would be checked >44k times on Mainnet, we should change it to [SubnetCount]bool and then we can just check the array at the subnet index instead of using bytes.Contains

Reply:
@moshe-blox There was a bug in my implementation, I don't need to use `bytes.Contains` actually, I can just check the index. I don't think we need to have an array in this PR just in this method, but we need to refactor subnets to have array as an underlying type